src/js/parsers/text.js

```diff
@@ -0,0 +1,84 @@
+import assert from 'mobiledoc-kit/utils/assert';
+import {
+  MARKUP_SECTION_TYPE,
+  LIST_SECTION_TYPE
+} from 'mobiledoc-kit/models/types';
+import {
+  DEFAULT_TAG_NAME as DEFAULT_MARKUP_SECTION_TAG_NAME
+} from 'mobiledoc-kit/models/markup-section';
+
+export const SECTION_BREAK = '\n';
+const UL_LI_REGEX = /^\* (.*)$/;
+const OL_LI_REGEX = /^\d.? (.*)$/;
```

Comment:
Shouldn't the `.` be escaped? Now I think it will match a digit followed by any character, not sure if that was intended. The line "12 people" would now result in an ordered list with `people`.